src/dotnet-structuring.library/Structuring.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using static dotnet_structuring.library.StructuringDelegate;
+
+namespace dotnet_structuring.library
+{
+    public class Structuring
+    {
+        public event StructuringHandler LogEvent;
+
+        private void WriteLog(string logs)
+        {
+            EventLogger log = new EventLogger
+            {
+                Logs = logs
+            };
+            LogEvent.Invoke(this, log);
+        }
+
+        public async Task RunStructuringAsync(string Output, IEnumerable<string> Directories, string NETCommand, string ProjectName)
```

Comment:
das sind doch auch alles Variablen :P

Reply:
also camelCase

Reply:
Paul hat doch aber gesagt, Private Variablen vorne klein und Public vorne Groß

Reply:
mhm aber die existieren ja trotzdem nur in der Methode

Reply:
also was soll ich da machen? 😄